lib/src/git_backend.rs

```diff
@@ -957,7 +957,7 @@ impl Backend for GitBackend {
             .map_err(|err| to_read_object_err(err, id))?;
         let target = String::from_utf8(blob.take_data())
             .map_err(|err| to_invalid_utf8_err(err.utf8_error(), id))?
-            .to_owned();
+            .clone();
```

Comment:
nit: this `.clone()` can be removed at all

Reply:
Done. One other example where seeing `.clone()` makes us question whether this is really needed or not, while seeing `to_owned()` might be legitimate if we had a slice.

Reply:
Just for clarification, I spotted it just because there was a diff. It doesn't matter (to me) whether it was `.clone()` or not.